test/Makefile

```diff
@@ -25,6 +29,110 @@ clean: # Cleanup the test environment.
 .PHONY: test
 test: # Run end-to-end tests.
 
+integration-test:
+	cd $(ROOT_DIR); go test -count=1 -tags=integration ./...
+
+e2e-test: clean-test-cluster test-cluster _e2e-test
+	$(MAKE) clean-test-cluster
+
+_e2e-test:
+	$(CHAINSAW) test --parallel 1 --test-dir $(ROOT_DIR)/test/e2e --selector $(TEST_SELECTOR)
+
+test-cluster: ctlptl kind helm
+	$(CTLPTL) apply -f $(ROOT_DIR)/test/ctlptl-config.yaml
+	kubectl create -k github.com/kubernetes-sigs/container-object-storage-interface-api
+	kubectl create -k github.com/kubernetes-sigs/container-object-storage-interface-controller
+	$(HELM) install linode-cosi-driver \
```

Comment:
Do we need custom build or we are happy with the default image provided by Helm chart?

Reply:
I would love to have there a custom image.